src/components/ProfileHeader/ProfileHeaderMenuItem/ProfileHeaderMenuItem.tsx

```diff
@@ -7,37 +7,48 @@ import { Icon, IconNames } from "../../Icon/Icon";
 interface ProfileHeaderMenuItemProps {
   icon: IconNames;
   badgeCount: number;
+  srText: string;
   className?: string;
+  popupOpen?: boolean;
   onClick?: MouseEventHandler<HTMLButtonElement> | undefined;
 }
 
 export const ProfileHeaderMenuItem = ({
   icon,
   badgeCount,
   className,
+  popupOpen,
+  srText,
   onClick,
 }: ProfileHeaderMenuItemProps & JSX.IntrinsicElements["button"]) => {
   return (
-    <button
-      className={`profile-header-menu-item ${className}`}
-      onClick={onClick}>
-      {(className || "").indexOf("user-profile") > -1 ? (
-        <>
-          <svg
-            xmlns="http://www.w3.org/2000/svg"
-            width="36"
-            height="36"
-            viewBox="0 0 40 40"
-            fill="none"
-            className="user-image"
-            stroke="#ddd">
-            <circle cx="20" cy="20" r="20"></circle>
-          </svg>
-          <Icon name="chevron-down" className="chevron-down" />
-        </>
-      ) : (
-        <Icon name={icon} hasBadge={badgeCount > 0} />
-      )}
-    </button>
+    <>
+      <span className="sr-only">{srText}</span>
+
+      <button
+        className={`profile-header-menu-item ${className}`}
+        onClick={onClick}>
+        {(className || "").indexOf("user-profile") > -1 ? (
```

Comment:
Taking a closer look at this and wondering if can remove this condition by moving the user-profile specific stuff into its own component.  Or do some polymorphism on menu items.

Reply:
Another thought: this profile button should always be rendered, regardless of what menu items are passed in.  I think it's an assumed thing.  Especially since the `ProfileHeader` component requires a `profileHeaderPopupOpen` boolean